crates/apollo-compiler/src/schema/mod.rs

```diff
@@ -831,6 +871,36 @@ impl From<Node<InputObjectType>> for ExtendedType {
     }
 }
 
+impl Implementers {
+    pub fn iter(&self) -> impl Iterator<Item = &'_ Name> {
+        self.objects.iter().chain(&self.interfaces)
+    }
+}
+
+impl IntoIterator for Implementers {
+    type Item = Name;
+    type IntoIter = std::iter::Chain<
```

Comment:
I think I have to write out the whole type here, right? There's no such thing as `type IntoIter = impl Iterator`

Reply:
Right, or at least there isn’t yet.
Maybe the `iter` method is enough?